core/http/client/src/main/java/org/eclipse/rdf4j/http/client/RDF4JProtocolSession.java

```diff
@@ -606,7 +609,7 @@ public void getStatements(Resource subj, IRI pred, Value obj, boolean includeInf
 		pingTransaction();
 	}
 
-	public synchronized void beginTransaction(IsolationLevel isolationLevel)
+	public synchronized void beginTransaction(TransactionSetting... transactionSettings)
```

Comment:
This is backward incompatible in multiple ways.

Reply:
I presume this was just me trying to get things working before I started considering backwards incompatability.